presto-hive/src/test/java/com/facebook/presto/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -177,6 +179,21 @@ public void createPartitionedTable(HiveStorageFormat storageFormat)
     }
 
     @Test
+    public void testCreatePartitionedTableInvalidColumnOrdering()
+    {
+        try {
+            assertUpdate("" +
+                    "CREATE TABLE test_show_columns_partition_key\n" +
+                    "(grape bigint, apple varchar, orange bigint, pear varchar)\n" +
+                    "WITH (partitioned_by = ARRAY['apple'])");
+            fail("expected exception");
+        }
+        catch (RuntimeException e) {
+            assertEquals(e.getMessage(), "Partition keys must be the last columns in the table, and in the same order as the table properties ([apple])");
```

Comment:
Since this is only asserting the message, it can use `expectedExceptions` / `expectedExceptionsMessageRegExp`